<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatroomForm</class>
 <widget class="QWidget" name="ChatroomForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="RoomLayout" stretch="2,8">
     <item>
      <layout class="QVBoxLayout" name="UserPane">
       <item>
        <widget class="QLabel" name="usersOnlineLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>12</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>12</height>
          </size>
         </property>
         <property name="text">
          <string>Users</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="UserrSplitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="UserInfoTextBox" name="UserInfoTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>134</width>
            <height>80</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextBrowserInteraction</set>
          </property>
         </widget>
         <widget class="QListWidget" name="UserListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ChallengeButtonLayout">
         <item>
          <spacer name="ChallengeButtonLeftSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ChallengeButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string comment="ButtonChallenge">Challenge</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="ChallengeButtonRightSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ChatPane">
       <item>
        <widget class="ChatroomTextEdit" name="ChatTextEdit">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="EntryLayout">
         <item>
          <widget class="ChatTextEntry" name="EntryTextEdit">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="SendButton">
             <property name="text">
              <string comment="ButtonSend">Send</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SendPrivateButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string comment="SendPrivate">Send Private</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ChallengeLayout">
     <item>
      <widget class="QGroupBox" name="ChallengeGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Status</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="RulesLabel">
          <property name="text">
           <string>Rules</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="RulesComboBox">
          <item>
           <property name="text">
            <string>Tsu</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fever</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fever (15th)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Endless Fever VS</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="PlayersLabel">
          <property name="text">
           <string>Players</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="PlayersSpinbox">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="PrivateCheckBox">
          <property name="text">
           <string comment="PrivateRoom">Private Room</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="AutoRejectCheckBox">
          <property name="text">
           <string comment="ButtonAutoreject">Autoreject</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="ChallengeArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Challenge Area</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ChallengeIcon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ChallengeRulesLayout">
          <item>
           <widget class="QPushButton" name="ReviewRulesButton">
            <property name="text">
             <string comment="ButtonReviewRules">Review Rules</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ChallengeRulesLabel">
            <property name="text">
             <string>...
...
...</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ChallengeQuestionLayout">
          <item>
           <widget class="QLabel" name="ChallengeQuestionLabel">
            <property name="text">
             <string>...
...
...</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ChallengeQuestionResponseLayout">
            <item>
             <widget class="QPushButton" name="ChallengeYesButton">
              <property name="text">
               <string comment="ButtonYes">Yes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ChallengeNoButton">
              <property name="text">
               <string comment="ButtonNo">No</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatTextEntry</class>
   <extends>QPlainTextEdit</extends>
   <header>chattextentry.h</header>
  </customwidget>
  <customwidget>
   <class>UserInfoTextBox</class>
   <extends>QTextEdit</extends>
   <header>userinfotextbox.h</header>
  </customwidget>
  <customwidget>
   <class>ChatroomTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>chatroomtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
